Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1258,7 +1258,11 @@ def getCoordinates(self, glyfTable, round=noRound):
         """
 
         if self.numberOfContours > 0:
-            return self.coordinates, self.endPtsOfContours, self.flags
+            coordinates = self.coordinates
+            if round is not noRound:
```

Comment:
I suggest a comment

Reply:
added a comment, and made the changes more lightweight